plio/utils/covariance.py

```diff
@@ -37,12 +38,15 @@ def compute_covariance(lat, lon, rad, latsigma=10., lonsigma=10., radsigma=15.,
     """
     lat = math.radians(lat)
     lon = math.radians(lon)
-    
+
+    if semimajor_axis is None:
+        semimajor_axis = rad
+
```

Comment:
Is there a case where this is desirable? Or should semimajor just be a required param?

Reply:
ISIS changed from using the semimajor and semiminor radii for this to using the local radius. So, if the goal is to go into ISIS, then we want to use this.